README.md

```diff
@@ -1,4 +1,4 @@
-# Weave - the Docker network
+# Weave - weaving applications into containers
```

Comment:
Isn't it supposed to be "weaving containers into applications" (i.e., a transposition)